app.js

```diff
@@ -0,0 +1,45 @@
+function submitToAPI(e) {
+    e.preventDefault();
+
+    const phoneVal = document.getElementById("phone-input").value;
+    const topicVal = document.getElementById("topic-select").value;
+
+    const tel_regex = /[0-9]{10}/;
+    if (tel_regex.test(phoneVal)) {
+        var full_phone = '+1';
+        full_phone += phoneVal;
+        let data = {
+            number: full_phone,
+            topic: topicVal
+        };
+
+        $.ajax({
+            type: "POST",
+            url: "https://djjqpcoffj.execute-api.us-west-2.amazonaws.com/prod/",
+            dataType: "json",
+            crossDomain: "true",
```

Comment:
`"true"` != `true` (again, welcome to JavaScript [being goddamn insane](https://stackoverflow.com/questions/11363659/why-does-true-true-show-false-in-javascript)). 

This difference doesn't appear to make a difference for the purposes of this AJAX call, though. It's more of a nitpick comment :sweat_smile: 

```suggestion
            crossDomain: true,
```